test/srt/test_server_args.py

```diff
@@ -0,0 +1,24 @@
+import unittest
```

Comment:
Where should this test be added in the CI pipeline?

https://github.com/sgl-project/sglang/blob/main/.github/workflows/pr-test.yml

Reply:
https://github.com/sgl-project/sglang/blob/9999442756cf34a298933d2e072bd07493346d52/.github/workflows/pr-test.yml#L40-L76

Reply:
@kevin85421 Add it here https://github.com/sgl-project/sglang/blob/474317f2b606a79ac6811798c612d13b83f719fd/test/srt/run_suite.py#L21

This "minimal" suite will be run on CI

Reply:
Thanks! Added https://github.com/sgl-project/sglang/pull/1298/commits/3ecf5746be95a7786d48dc32d06910b14cb4e209.